ModernFlyouts/Flyout Helpers/LockKeysFlyoutHelper.cs

```diff
@@ -126,6 +132,11 @@ void ShowFlyout()
             }
         }
 
+        private void KeyRelased(Key key, int virtualKey)
```

Comment:
KeyReleased*

Reply:
Lines 183 and 192 need to be renamed as well.

Reply:
Oh, i didn't notice, Ty!